core/src/wallet/tezos/TezosLikeAccount2.cpp

```diff
@@ -206,11 +207,10 @@ namespace ledger {
                                           return seq.str();
                                       })
                     .recoverWith(getContext(), [] (const Exception &e) {
+                        // Avoid fmt error because of JSON string
+                        auto error = "Failed to broadcast raw transaction, reason " + e.getMessage();
                         return Future<std::string>::failure(
-                                make_exception(
-                                        api::ErrorCode::INCOMPLETE_TRANSACTION,
-                                        fmt::format("Failed to broadcast raw transaction, reason {}", e.getMessage())
-                                )
+                                Exception(api::ErrorCode::INCOMPLETE_TRANSACTION, error)
```

Comment:
What is the advantage to use `make_exception` over `Exception` ?

Reply:
Not going through `fmt` which is crashing if you don't escape braces ! :trollface:

Reply:
You cannot use here `make_exception(api::ErrorCode::INCOMPLETE_TRANSACTION, error)` ?

Reply:
Please read my comment above (in the code too) and `make_exception` code ! 👌